<a href="https://colab.research.google.com/github/glgunderson/AI-Perceptions/blob/main/AI_Perceptions.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

In [ ]:
!wget -O AIUU.xlsx "https://raw.githubusercontent.com/glgunderson/AI-Perceptions/main/data/AIUU.xlsx"

--2026-03-30 18:41:38--  https://raw.githubusercontent.com/glgunderson/AI-Perceptions/main/data/AIUU.xlsx
Resolving raw.githubusercontent.com (raw.githubusercontent.com)... 185.199.108.133, 185.199.109.133, 185.199.110.133, ...
Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|185.199.108.133|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 30368 (30K) [application/octet-stream]
Saving to: ‘AIUU.xlsx’

AIUU.xlsx           100%[===================>]  29.66K  --.-KB/s    in 0.001s  

2026-03-30 18:41:38 (36.6 MB/s) - ‘AIUU.xlsx’ saved [30368/30368]



In [ ]:
import pandas as pd
df = pd.read_excel("AIUU.xlsx")
df.head()

,Age,Education,Frequency,Job status,Field,Field_10_TEXT,Perceptions_1,Perceptions_2,Perceptions_3,Perceptions_4,...,Procedural Fairness_2,Outcome Fairness_1,Outcome Fairness_2,Outcome Fairness_3,Performance Pressure_1,Performance Pressure_2,Performance Pressure_3,Competency_1,Competency_2,Competency_3
0,23,2,4,5,2,NaN,4,4,4,4,...,4,4,4,5,5,4,3,3,4,5
1,21,1,5,5,2,NaN,4,4,4,5,...,3,4,3,4,3,4,4,4,3,3
2,20,1,2,5,"1,2,4",NaN,1,2,2,3,...,2,2,3,2,2,3,3,1,1,3
3,23,1,4,1,7,NaN,4,4,2,3,...,3,3,3,3,3,3,3,3,3,3
4,30,3,5,5,1,NaN,4,4,4,4,...,4,4,4,4,4,4,4,4,4,4


In [ ]:
df.info()
df.describe()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 122 entries, 0 to 121
Data columns (total 49 columns):
 #   Column                  Non-Null Count  Dtype 
---  ------                  --------------  ----- 
 0   Age                     122 non-null    int64 
 1   Education               122 non-null    int64 
 2   Frequency               122 non-null    int64 
 3   Job status              122 non-null    int64 
 4   Field                   122 non-null    object
 5   Field_10_TEXT           7 non-null      object
 6   Perceptions_1           122 non-null    int64 
 7   Perceptions_2           122 non-null    int64 
 8   Perceptions_3           122 non-null    int64 
 9   Perceptions_4           122 non-null    int64 
 10  Perceptions_5           122 non-null    int64 
 11  AI Expectations_1       122 non-null    int64 
 12  AI Expectations_2       122 non-null    int64 
 13  AI Expectations_3       122 non-null    int64 
 14  AI Expectations_4       122 non-null    int64 
 15  AI Exp

,Age,Education,Frequency,Job status,Perceptions_1,Perceptions_2,Perceptions_3,Perceptions_4,Perceptions_5,AI Expectations_1,...,Procedural Fairness_2,Outcome Fairness_1,Outcome Fairness_2,Outcome Fairness_3,Performance Pressure_1,Performance Pressure_2,Performance Pressure_3,Competency_1,Competency_2,Competency_3
count,122.000000,122.000000,122.000000,122.000000,122.000000,122.000000,122.000000,122.000000,122.000000,122.000000,...,122.000000,122.000000,122.000000,122.000000,122.000000,122.000000,122.000000,122.000000,122.000000,122.000000
mean,22.459016,1.368852,3.877049,2.483607,3.860656,3.704918,3.622951,4.188525,4.467213,4.131148,...,3.549180,2.573770,3.885246,2.450820,3.827869,3.877049,3.844262,3.901639,3.819672,3.893443
std,3.072532,0.645401,1.041120,1.745045,0.846144,0.906255,0.947619,0.806297,0.563115,0.781510,...,0.717001,1.019788,0.844982,0.988383,0.849819,0.838908,0.890914,0.885231,0.962232,0.779991
min,19.000000,1.000000,1.000000,1.000000,1.000000,1.000000,1.000000,1.000000,3.000000,2.000000,...,1.000000,1.000000,2.000000,1.000000,1.000000,2.000000,1.000000,1.000000,1.000000,1.000000
25%,21.000000,1.000000,3.000000,1.000000,4.000000,3.000000,3.000000,4.000000,4.000000,4.000000,...,3.000000,2.000000,3.250000,2.000000,3.000000,4.000000,3.000000,4.000000,3.000000,4.000000
50%,21.000000,1.000000,4.000000,2.000000,4.000000,4.000000,4.000000,4.000000,4.500000,4.000000,...,4.000000,2.000000,4.000000,2.000000,4.000000,4.000000,4.000000,4.000000,4.000000,4.000000
75%,23.000000,2.000000,5.000000,5.000000,4.000000,4.000000,4.000000,5.000000,5.000000,5.000000,...,4.000000,3.000000,4.000000,3.000000,4.000000,4.000000,4.000000,4.000000,4.000000,4.000000
max,37.000000,3.000000,5.000000,5.000000,5.000000,5.000000,5.000000,5.000000,5.000000,5.000000,...,5.000000,5.000000,5.000000,5.000000,5.000000,5.000000,5.000000,5.000000,5.000000,5.000000


In [ ]:
!pip install pingouin

# Qualtrics Survey
**Age** What is your age?

**Education** What is your educational status?
1. Final-year bachelor student (graduating within a year)
2. Recent bachelor graduate
3. Recent masters graduate
4. Recent PhD graduate

**Frequency** How often do you use Al tools?
1. Never
2. Sometimes (monthly)
3. About half the time (weekly)
4. Most of the time (multiple times a week)
5. Daily

**Job status** What is your current job status?
1. Not yet applying
2. Applying
3. Interviewing
4. Offer(s) Received
5. Employed

**Field** Which industry are you interested in working in?
1. Technology / IT / Data
2. Finance / Banking / Accounting
3. Consulting / Business Services
4. Marketing / Media / Communications
5. Law / Legal Services
6. Government / Public Sector / NGOs
7. Healthcare / Life Sciences
8. Education / Research / Academia
9. Engineering / Manufacturing
10. Other

### **LIKERT SCALE**

5 = Strongly agree

4 = Agree

3 = Neither agree nor disagree

2 = Disagree

1 = Strongly Disagree

***5-point Likert scale is used to measure AI Perceptions, Performance Pressure, and Competency Expectations***

### **Perceptions of AI in the Workplace**

*Please indicate to which extent do you agree with the following statements:*

1. **Perceptions_1**
I feel confident in my understanding of Al tools.

2. **Perceptions_2**
I feel confident in my understanding of how Al tools are used in the workplace.

3. **Perceptions_3**
I believe employers rely on Al tools to evaluate current and future employees.

4. **Perceptions_4**
The future workplace will rely heavily on Al-supported systems.

5. **Perceptions_5**
Al technologies are becoming increasingly common in the workplace.

In [ ]:
PP = df[['Performance Pressure_1', 'Performance Pressure_2', 'Performance Pressure_3']]
CE = df[['Competency_1', 'Competency_2', 'Competency_3']]

### **Performance Pressure (PP)**

1. **Performance Pressure_1 (PP1)**
Al technologies will increase expectations for employee productivity.

2. **Performance Pressure_2 (PP2)**
Al technologies will require employees to complete tasks more quickly.

3. **Performance Pressure_3 (PP3)**
Al technologies will raise performance standards in entry-level roles.


### **Competency Expectations (CE)**
1. **Competency_1 (CE1)**
Because of Al technologies, employers will expect workers to have stronger digital or technical skills.

2. **Competency_2 (CE2)**
Because of Al technologies, entry-level roles will require more advanced skills.

3. **Competency_3 (CE3)**
Because of Al technologies, employers will expect workers to continuously update their skills.



# **Cronbach's Alpha**

## Performance Pressure (PP)

Cronbach’s alpha for the Performance Pressure scale indicated good internal consistency **(α = 0.81)**, suggesting that the items reliably capture a common underlying construct related to perceived AI-driven performance expectations.

Item-level analysis showed that the first two items (PP1, PP2) demonstrated strong internal consistency (α = 0.81), while pairwise combinations involving the third item (PP3) were slightly lower (α = 0.68–0.72). This indicates that, although all items contribute to the construct, one item reflects a somewhat distinct aspect of performance pressure.

Overall, the scale can be considered reliable, with minor variation across items reflecting nuanced dimensions of perceived pressure.


## Competency Expectations (CE)

Cronbach’s alpha for the Competency Expectations scale indicated marginal internal consistency **(α = 0.69)**. While close to the commonly accepted threshold of 0.70, this suggests some variability in how consistently the items measure the construct.

Item-level analysis revealed that the first two items (CE1, CE2) showed moderate consistency (α = 0.69), whereas pairwise combinations involving the third item (CE3) were notably lower (α = 0.48–0.59). This suggests that the third item is less aligned with the others.

These findings indicate that the scale may capture related but distinct dimensions of competency expectations, particularly differentiating between immediate skill requirements and longer-term expectations for continuous skill development.

In [ ]:
import pingouin as pg

In [ ]:
PP = df[['Performance Pressure_1', 'Performance Pressure_2', 'Performance Pressure_3']]

pg.cronbach_alpha(PP)

(np.float64(0.8056367874549691), array([0.737, 0.859]))

**Performance Pressure (PP)**

Cronbach's alpha for the Performance Pressure scale demonstrated good internal consistency **(α = 0.81)**, indicating that the items reliably capture a common underlying construct.

In [ ]:
CE = df[['Competency_1', 'Competency_2', 'Competency_3']]

pg.cronbach_alpha(CE)

(np.float64(0.6923476372562903), array([0.584, 0.776]))

**Competency Expectations (CE)**

Cronbach’s alpha for the Competency Expectations scale demonstrated marginal internal consistency **(α = 0.69)**. While close to the commonly accepted threshold of 0.70, the relatively wide confidence interval suggests variability in item coherence.

*Item-level analysis indicated that the third item exhibited weaker alignment with the other indicators, suggesting that the item related to continuous skill updating may capture a distinct, more long-term dimension of perceived expectations.*

In [ ]:
PP12 = df[['Performance Pressure_1', 'Performance Pressure_2']]

pg.cronbach_alpha(PP12)

(np.float64(0.807500566508045), array([0.725, 0.865]))

In [ ]:
PP13 = df[['Performance Pressure_1', 'Performance Pressure_3']]

pg.cronbach_alpha(PP13)

(np.float64(0.6750740082889277), array([0.535, 0.773]))

In [ ]:
PP23 = df[['Performance Pressure_2', 'Performance Pressure_3']]

pg.cronbach_alpha(PP23)

(np.float64(0.7173030056864336), array([0.596, 0.802]))

In [ ]:
CE12 = df[['Competency_1', 'Competency_2']]

pg.cronbach_alpha(CE12)

(np.float64(0.6929770043505283), array([0.561, 0.785]))

In [ ]:
CE13 = df[['Competency_1', 'Competency_3']]

pg.cronbach_alpha(CE13)

(np.float64(0.4817318704052609), array([0.259, 0.638]))

In [ ]:
CE23 = df[['Competency_2', 'Competency_3']]

pg.cronbach_alpha(CE23)

(np.float64(0.5880115956485148), array([0.411, 0.712]))

Pairwise reliability analysis indicated that items within the Performance Pressure scale were generally consistent, although one item showed slightly weaker alignment (Performance Pressure_3), suggesting minor variation in how pressure is perceived.

In contrast, the Competency Expectations scale showed greater variability at the item level. The item related to continuous skill updating (Cometency_3) demonstrated weaker associations with other items, indicating that it may capture a more distinct, long-term dimension of competency expectations.

## Descriptive Statistics

Descriptive statistics were calculated for the composite variables Performance Pressure (PP) and Competency Expectations (CE).

The mean score for Performance Pressure was M = 3.85 (SD = 0.73), while the mean score for Competency Expectations was M = 3.87 (SD = 0.69), indicating generally positive agreement with both constructs.


Both variables showed similar distributions, with **median values of 4.00 and interquartile ranges between 3.67 and 4.33**.

The similarity in central tendency and dispersion suggests that respondents perceive both increased performance demands and competency expectations as comparable outcomes of AI integration. Median scores of 4.00 further indicate a tendency toward agreement across the sample.

These results indicate that, on average, respondents tended to **agree that AI technologies are associated with increased performance pressure and competency expectations in the workplace.** The relatively moderate standard deviations suggest some variation in responses, but no extreme dispersion.

In [ ]:
# Composite Variables
df['PP_score'] = PP.mean(axis=1)
df['CE_score'] = CE.mean(axis=1)

In [ ]:
df[['PP_score', 'CE_score']].describe()

,PP_score,CE_score
count,122.000000,122.000000
mean,3.849727,3.871585
std,0.729923,0.691609
min,1.666667,1.666667
25%,3.666667,3.666667
50%,4.000000,4.000000
75%,4.333333,4.333333
max,5.000000,5.000000


**MEAN**

PP mean = 3.85

CE mean = 3.87


Students perceive both performance pressure and competency excpectations at equally high levels

**STANDARD DEVIATION**

PP std = 0.73

CE std = 0.70

People are responding consistently across both constructs.

## Correlation Analysis
Pearson correlation analysis revealed a moderate positive relationship between Performance Pressure and Competency Expectations **(r = 0.53)**, suggesting that **higher perceived performance pressure is associated with higher competency expectations**.


While the strength of the correlation indicates a meaningful association, it also confirms that the two constructs are related but not identical, supporting their treatment as distinct dimensions of perceived AI-related workplace expectations.

In [ ]:
df[['PP_score', 'CE_score']].corr()

,PP_score,CE_score
PP_score,1.000000,0.528989
CE_score,0.528989,1.000000


**Correlation (r = 0.53)**
- Moderate positive correlation.

Meaning:
- When someone perceives more performance pressure, they ALSO tend to perceive more competency escalation.

But, not super high (not like 0.8+).

So, they are related, but NOT the same thing.

Core conceptual finding:
- AI is not just increasing one type of expectation, but multiple linked but distinct pressures.

**Findings**

Both performance pressure (M = 3.85) and competency escalation (M = 3.87) were perceived at similarly high levels among respondents.

The two constructs were moderately correlated (r = 0.53), suggesting that while related, they represent distinct dimensions of perceived workplace expectations.

AI makes people feel like they must:
- work harder/faster (PP)
- be more skilled (CE)

And those pressures:
- move together
- but are not identical


## IV Construct Refinement

In [ ]:
IV5 = df[['Perceptions_1','Perceptions_2','Perceptions_3','Perceptions_4','Perceptions_5']]
df['IV5_score'] = IV5.mean(axis=1)

pg.cronbach_alpha(IV5)

(np.float64(0.5620418360526405), array([0.426, 0.674]))

Initial reliability analysis indicated low internal consistency (α = 0.56) when combining all perception items. Items reflecting personal confidence were conceptually distinct from perceived AI integration and were therefore excluded. The final composite variable focused on perceived AI prevalence and use in the workplace.

In [ ]:
IV12 = df[['Perceptions_1','Perceptions_2']]
df['IV12_score'] = IV12.mean(axis=1)

pg.cronbach_alpha(IV12)

(np.float64(0.6712533302104986), array([0.53, 0.77]))

In [ ]:
IV145 = df[['Perceptions_1','Perceptions_4','Perceptions_5']]
df['IV145_score'] = IV145.mean(axis=1)

pg.cronbach_alpha(IV145)

(np.float64(0.5160123557831934), array([0.346, 0.648]))

In [ ]:
IV1345 = df[['Perceptions_1','Perceptions_3','Perceptions_4','Perceptions_5']]
df['IV1345_score'] = IV1345.mean(axis=1)

pg.cronbach_alpha(IV1345)

(np.float64(0.5395828450902276), array([0.39, 0.66]))

In [ ]:
IV345 = df[['Perceptions_3','Perceptions_4','Perceptions_5']]
df['IV345_score'] = IV345.mean(axis=1)

pg.cronbach_alpha(IV345)

(np.float64(0.5863419166445449), array([0.441, 0.699]))

In [ ]:
# REFINE IV - AI Prevalence / Adoption - Perceived AI presence / integration in the workplace

IV = df[['Perceptions_4','Perceptions_5']]
df['IV_score'] = IV.mean(axis=1)

pg.cronbach_alpha(IV)

(np.float64(0.7384696942922779), array([0.626, 0.817]))

Perceived AI integration was operationalized using two items reflecting the increasing prevalence and future dominance of AI in the workplace. These items demonstrated acceptable internal consistency (α = 0.74).

Normally, 2-item scales are weaker.

BUT:
- alpha = 0.74
- items are conceptually tight
- study is not psychometrics-heavy

So, this is defensible.

**Perceived AI integration was operationalized using two items reflecting the increasing prevalence and future dominance of AI in the workplace. These items demonstrated acceptable internal consistency (α = 0.73).**

DEFINITION
Perceived AI integration was operationalized as the extent to which respondents believe AI is understood, prevalent, and actively used in workplace contexts.

Measuring... how much students perceive AI as present, important, and influential in the workplace

IV → perceived AI integration

DV1 → performance pressure

DV2 → competency escalation

In [ ]:
IV = df[['Perceptions_4', 'Perceptions_5']]
df['IV_score'] = IV.mean(axis=1)

In [ ]:
df[['IV_score', 'PP_score', 'CE_score']].corr()

,IV_score,PP_score,CE_score
IV_score,1.000000,0.469564,0.311436
PP_score,0.469564,1.000000,0.528989
CE_score,0.311436,0.528989,1.000000


Initial reliability analysis indicated that the full 5-item scale lacked internal consistency (α = .57). After refinement, a 2-item scale capturing perceived AI integration demonstrated acceptable reliability (α = .73) and was retained for analysis.

Core Finding:
AI is not just increasing one type of expectation, but multiple linked but distinct pressures.

Coorelation Insight:
PP ↔ CE = 0.55 → moderate
IV ↔ PP = 0.46 → solid
IV ↔ CE = 0.28 → weaker

Key pattern:
	•	AI perception → stronger effect on performance pressure (0.46)
	•	AI perception → weaker effect on competency escalation (0.28)

⸻


Students are thinking:

“AI means I have to work harder/faster RIGHT NOW”


More than:

“AI means I need to become more skilled LONG-TERM”


**Perceived AI integration was positively associated with both performance pressure and competency escalation. However, the relationship was notably stronger for performance pressure (r = .46) than for competency escalation (r = .28), suggesting that students interpret AI primarily as increasing immediate productivity demands rather than long-term skill requirements.**

**While the two constructs were moderately correlated (r = .55), indicating they co-occur, the difference in their relationship with AI perception suggests that they represent distinct dimensions of perceived workplace expectations.**

Work intensification → immediate pressure

Employability → long-term skills


And data shows:
students feel the first more strongly than the second

This:
PP mean ≈ 3.85
CE mean ≈ 3.87

Combined with:
low variance + consistent responses

Means:
Students broadly AGREE AI raises expectations

Not divided. Not controversial.

That’s a consensus perception, which strengthens argument.


In [ ]:
import statsmodels.api as sm

# Independent variable
X = df['IV_score']
X = sm.add_constant(X)

# Model 1: Performance Pressure
model_PP = sm.OLS(df['PP_score'], X).fit()

# Model 2: Competency Escalation
model_CE = sm.OLS(df['CE_score'], X).fit()

print(model_PP.summary())
print(model_CE.summary())

                            OLS Regression Results                            
Dep. Variable:               PP_score   R-squared:                       0.220
Model:                            OLS   Adj. R-squared:                  0.214
Method:                 Least Squares   F-statistic:                     33.94
Date:                Mon, 30 Mar 2026   Prob (F-statistic):           4.87e-08
Time:                        18:41:45   Log-Likelihood:                -119.01
No. Observations:                 122   AIC:                             242.0
Df Residuals:                     120   BIC:                             247.6
Df Model:                           1                                         
Covariance Type:            nonrobust                                         
                 coef    std err          t      P>|t|      [0.025      0.975]
------------------------------------------------------------------------------
const          1.4539      0.415      3.500      0.0

FINDINGS
Perceived AI integration was positively associated with both performance pressure (β = 0.55, p < .001) and competency escalation (β = 0.31, p = .003). However, the relationship was substantially stronger for performance pressure, indicating that students interpret AI primarily as increasing immediate productivity demands rather than long-term skill requirements. The model explained 21.4% of the variance in performance pressure, compared to 7.8% for competency escalation.

⸻

Interpretation

These findings suggest that students perceive AI not only as a driver of increased expectations, but specifically as intensifying short-term performance demands more than long-term skill development. This aligns more closely with work intensification perspectives than with traditional human capital or employability frameworks.